system/forms.py

```diff
@@ -56,3 +56,8 @@ class Meta:
             'D_from': forms.SelectDateWidget(years=year_range),
             'D_to': forms.SelectDateWidget(years=year_range),
         }
+
+
+class SearchForm(forms.Form):
+    patient = forms.ModelChoiceField(queryset=Patient.objects.all())
+    window = forms.IntegerField(min_value=1)
```

Comment:
Window can be valued 0 for same day inspections.
So the min_value should be 0.

Reply:
True.
I will change this later